test/e2e/network_test.go

```diff
@@ -29,9 +29,9 @@ import (
 func testNetwork(t *testing.T) {
 	tc, err := NewTestContext()
 	require.NoError(t, err)
-	err = tc.waitForWindowsNodes(gc.numberOfMachineNodes, false, false, false)
+	err = tc.waitForConfiguredWindowsNodes(gc.numberOfMachineNodes, false, false)
```

Comment:
This test needs a node that is schedulable. How are you guaranteeing that now that you have moved to just checking if the node has been configured? This becomes especially important if I am just running the networking test without running the create tests but bringing up nodes outside of the test suite.

Reply:
+1, might have to run testNodesBecomeReady() here instead

Reply:
> How are you guaranteeing that now

I am not sure why this needs to be guaranteed within this test?
The test itself will fail if a node is not schedulable. This is the networking test, retesting node statuses that we test within the creation/validation tests is overlapping responsibilities.

Reply:
To elaborate, schedulable is a part of the node spec that is applied by WMCO or a user. IMO checking for that would be similar to checking for any taint that isn't tolerated.

Reply:
>I am not sure why this needs to be guaranteed within this test?
The test itself will fail if a node is not schedulable. This is the networking test, retesting node statuses that we test within the creation/validation tests is overlapping responsibilities.

You are assuming the creation test suite was run prior. What if I brought up the nodes manually and then manually ran the just networking test suite? Will `loadExistingNodes()` take care of that?

Reply:
This is still being tested, its just not a separate check. Specifically, it will be tested by the failure of the windows server deployments to be scheduled on the node. The pod events will state that no nodes meet the scheduling criteria.